tests/tck/features/insert/Insert.feature

```diff
@@ -396,29 +432,47 @@ Feature: Insert string vid of vertex and edge
     # insert multi version vertex
     When executing query:
       """
-      INSERT VERTEX person(name, age) VALUES "Tony":("Tony", 18), "Mack":("Mack", 19);
-      INSERT VERTEX person(name, age) VALUES "Mack":("Mack", 20);
-      INSERT VERTEX person(name, age) VALUES "Mack":("Mack", 21)
+      INSERT VERTEX person(name, age) VALUES <vid_Tony>:("Tony", 18), <vid_Mack>:("Mack", 19);
+      INSERT VERTEX person(name, age) VALUES <vid_Mack>:("Mack", 20);
+      INSERT VERTEX person(name, age) VALUES <vid_Mack>:("Mack", 21)
       """
     Then the execution should be successful
     # insert multi version edge
     When executing query:
       """
-      INSERT EDGE schoolmate(likeness, nickname) VALUES "Tony"->"Mack"@1:(1, "");
-      INSERT EDGE schoolmate(likeness, nickname) VALUES "Tony"->"Mack"@1:(2, "");
-      INSERT EDGE schoolmate(likeness, nickname) VALUES "Tony"->"Mack"@1:(3, "");
+      INSERT EDGE schoolmate(likeness, nickname) VALUES <vid_Tony>-><vid_Mack>@1:(1, "");
+      INSERT EDGE schoolmate(likeness, nickname) VALUES <vid_Tony>-><vid_Mack>@1:(2, "");
+      INSERT EDGE schoolmate(likeness, nickname) VALUES <vid_Tony>-><vid_Mack>@1:(3, "");
       """
     Then the execution should be successful
     # get multi version result through go
     When executing query:
       """
-      GO FROM "Tony" OVER schoolmate YIELD $$.person.name, $$.person.age, schoolmate.likeness
+      GO FROM <vid_Tony> OVER schoolmate YIELD $$.person.name, $$.person.age, schoolmate.likeness
       """
     Then the result should be, in any order:
       | $$.person.name | $$.person.age | schoolmate.likeness |
       | 'Mack'         | 21            | 3                   |
     Then drop the used space
 
+    Examples: Vertical
```

Comment:
Do you want to use the same feature file for int and string id tests ?

Reply:
yeah